src/server/snap.rs

```diff
@@ -488,6 +488,7 @@ impl<R: RaftExtension + 'static> Runnable for Runner<R> {
                 let recving_count = self.recving_count.clone();
                 recving_count.fetch_add(1, Ordering::SeqCst);
                 let limiter = self.snap_mgr.limiter().clone();
+                let snap_mgr_v1 = self.snap_mgr.clone();
```

Comment:
I think it would be better to have SNAP_TASK_COUNTER_STATIC.recv2.inc() in L484 for easier troubleshooting.

Reply:
Also in server.rs, under Server definition, it's better to have a general flag instead of tiflash_engine.
For example, v2_compatible_learner: bool. And TiFlash will have it true in its config.

Reply:
Good suggestion and done.